docs/source/onnxruntime/usage_guides/gpu.mdx

```diff
@@ -0,0 +1,225 @@
+)
+print("providers:", ort_model.providers)
+
+tokenizer = AutoTokenizer.from_pretrained("philschmid/tiny-bert-sst2-distilled")
+inp = tokenizer("expectations were low, actual enjoyment was high", return_tensors="pt", padding=True)
+
+result = ort_model(**inp)
+print(result)
+```
+
+In case the printed result has `providers: ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider']` and no error is raised, congratulations, the installation is successful!
+
+In case the printed result has only `providers: ['CUDAExecutionProvider', 'CPUExecutionProvider']`, or the warning `Failed to create TensorrtExecutionProvider. Please reference https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements to ensure all dependencies are met.` is raised, something is off in your installation.
+
+### Use TensorRT execution provider with floating-point models
+
+For non-quantized models, the use is straightforward, by simply using the `provider` argument in `ORTModel.from_pretrained()`. For example:
+
+```python
+from optimum.onnxruntime import ORTModelForSequenceClassification
+
+ort_model = ORTModelForSequenceClassification.from_pretrained(
+    "distilbert-base-uncased-finetuned-sst-2-english",
+    from_transformers=True,
+    provider="TensorrtExecutionProvider",
+)
+```
+
+[As previously for `CUDAExecutionProvider`](#use-cudaexecutionprovider-with-floatingpoint-model), by passing the session option `log_severity_level = 0` (verbose), we can check in the logs whether all nodes are indeed placed on the TensorRT execution provider or not:
+
+```
+2022-09-22 14:12:48.371513741 [V:onnxruntime:, session_state.cc:1188 VerifyEachNodeIsAssignedToAnEp] All nodes have been placed on [TensorrtExecutionProvider]
+```
+
+### Use TensorRT execution provider with quantized models
+
+When it comes to quantized models, TensorRT supports only models that use **static** quantization with **symmetric quantization** for weights and activations. Thus, to be able to consume with TensorRT models quantized through 🤗 Optimum, the following configuration needs to be passed when doing [static quantization](quantization#static-quantization-example):
+
+```python
+qconfig = QuantizationConfig(
+    ...,
+    is_static=True,
+    activations_symmetric=True,
+    weights_symmetric=True,
+    qdq_dedicated_pair=True,
+    qdq_add_pair_to_weight=True
+)
+```
+
+The parameter `qdq_dedicated_pair=True` is required by TensorRT, that expects a single node after each `QuantizeLinear` + `DequantizeLinear` (QDQ) pair.
+
+The parameter `qdq_add_pair_to_weight=True` is also required by TensorRT, that consumes a graph where the weights are stored in float32 with a QDQ pair. Normally, weights would be stored in fixed point 8-bits format and only a `DequantizeLinear` would be applied on the weights. As such, the storage savings from quantization can not be leveraged when we expect to later use the quantized `.onnx` model with TensorRT.
```

Comment:
```suggestion
The parameter `qdq_add_pair_to_weight=True` is also required by TensorRT, that since it consumes a graph where the weights are stored in float32 with a QDQ pair. Normally, weights would be stored in fixed point 8-bits format and only a `DequantizeLinear` would be applied on the weights. As such, the storage savings from quantization can not be leveraged when we expect to later use the quantized ONNX model with TensorRT.
```